src/horizon_api.ts

```diff
@@ -88,6 +88,17 @@ export namespace Horizon {
     ? BalanceLineAsset<T>
     : BalanceLineNative | BalanceLineAsset;
 
+  export interface AssetAccounts {
+    authorized: number;
+    authorized_to_maintain_liabilities: number;
+    unauthorized: number;
+  }
+  export interface AssetBalances {
+    authorized: string;
+    authorized_to_maintain_liabilities: string;
+    unauthorized: string;
+  }
+
```

Comment:
Looking at this closer, I'm a little confused about where this is coming from. What's the endpoint, here? 

In either case, I believe we also need a `clawbackEnabled` field here.

Reply:
So, these are new fields on `/assets`. ClawbackEnabled would go on the trustline flags (which these are not). These are sums of the number of accounts and the summed balances of those accounts broken down by the trustline state.

Reply:
Ah, gotcha, I see the corresponding fields e.g. [here](https://horizon-testnet.stellar.org/assets?asset_issuer=GCF3C5LRTWYU7UDSF2PO6VOPVPNU5MT6VJDQE4754QNHGOQS2R5CAQ5B). We *do* need to update the `flags` field, but you're right that this is out of scope here.